src/index.html

```diff
@@ -11,12 +11,128 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.google.com/specimen/Roboto"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    
+    <header class="header">
+
+      <a
+        class="logo"
+        href="moyo"
```

Comment:
Don't use spaces in the 'href' property of the 'a' tag. Anchor links should start with the '#' symbol if they are meant to be placeholders.